modules/ROOT/pages/guide-import-csv.adoc

```diff
@@ -74,27 +74,27 @@ However, if you need to access files in other locations, you can find out which
 Default import folder paths shown in link:/developer/kb/import-csv-locations/[this article^].
 
 .Examples
-[source, cypher, role= nocopy noplay]
+[source,cypher.nocopy.noplay]
 ----
 //Example 1 - file directly placed in import directory (import/data.csv)
-LOAD CSV FROM "file:///data.csv"
+LOAD CSV FROM "file:///data.csv";
 
 //Example 2 - file placed in subdirectory within import directory (import/northwind/customers.csv)
-LOAD CSV FROM "file:///northwind/customers.csv"
+LOAD CSV FROM "file:///northwind/customers.csv";
 ----
 
 *Web-hosted files* can be referenced directly with their URL, like `+https://host/path/data.csv+`.
 However, permissions must be set so that an external source can read the file.
 For more information about access related to online file imports, see this link:/developer/kb/import-csv-locations/[knowledge base article^].
 
 .Examples
-[source, cypher, role= nocopy noplay]
+[source,cypher.nocopy.noplay]
 ----
 //Example 1 - website
-LOAD CSV FROM 'https://data.neo4j.com/northwind/customers.csv'
+LOAD CSV FROM 'https://data.neo4j.com/northwind/customers.csv';
```

Comment:
These are only fragments of a query rather than an entire query, so the semi colons shouldn't be appended